packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -100,7 +101,10 @@ describe( 'Post Editor Performance', () => {
 		let i = 5;
 		while ( i-- ) {
 			await page.reload();
-			await page.waitForSelector( '.wp-block' );
+			await page.waitForSelector( '.edit-post-layout', {
+				timeout: 120000,
+			} );
+			await canvas().waitForSelector( '.wp-block', { timeout: 120000 } );
```

Comment:
Copied this approach from the site editor performance tests: 

https://github.com/WordPress/gutenberg/blob/b1234a731fad3a4e1f6b5a196efd47f765cc1e82/packages/e2e-tests/specs/performance/site-editor.test.js#L94-L97

Not sure if the large timeout is necessary. Should be harmless though.

Reply:
personally I've raised questions about them too and I think we'd benefit but cutting out earlier. in cases like these we have to wait two extra minutes before calling it a failure, but the threshold is far earlier than two minutes where we know something went wrong.